CMakeLists.txt

```diff
@@ -19,6 +19,12 @@ else()
   set( CMAKE_CXX_EXTENSIONS OFF )
 endif()
 
+option(LOAD_QUERY_TXID_PLUGIN "Build with query_txid plugin." ON)
```

Comment:
Yes, i just change `OFF` to  `ON` for testing ,i will modify it

Reply:
> IMO default should remain OFF because of the external dependency. Opinions @abitmore?

I agree to set the default to OFF so far. Perhaps change it to ON when we're sure it's stable and we want typical nodes to have it.

By the way, BitShares 0.x has [code using leveldb](https://github.com/bitshares/bitshares1-core/search?q=leveldb&unscoped_q=leveldb) but it's never an external dependency, why it's now an external dependency?